package-lock.json

```diff
@@ -8515,11 +8515,6 @@
       "integrity": "sha512-YJhbp0sz3egFFKl3BcCNPQKzuGFOP4PACcsifhK6ROGnJUW9ViYLuLybQ9GQZm7Zejy3tkGuiXYMq3GiyGkU4g==",
       "dev": true
     },
-    "@types/mime-types": {
```

Comment:
Again, not a big deal, but since this requires changes we may as well roll this file back 😄 

It's as simple as just checking out the version of the file:
```
git checkout origin/develop -- package-lock.json
```

Reply:
Ahhh, tried to manually revert changes and didn't quite catch it all. This code snippet is super handy, will definitely make use of this.🙏